src/Plugin.php

```diff
@@ -188,6 +188,7 @@ public static function init() {
 
     // Enqueues plugin scripts and styles.
     add_action('wp_enqueue_scripts', __CLASS__ . '::wp_enqueue_scripts');
+    add_action('wp_enqueue_scripts', __CLASS__ . '::wp_enqueue_elementor_script', 999);
```

Comment:
The priority needs to be less than the default (10) in order to be triggered before Elementor:
```suggestion
    add_action('wp_enqueue_scripts', __CLASS__ . '::wp_enqueue_elementor_script', 9);
```

Note that Elementor also appears to invoke an own hook when enqueueing its own scripts, so you could try this:
```suggestion
    add_action('elementor/frontend/before_enqueue_scripts', __CLASS__ . '::wp_enqueue_elementor_script');
```

Reply:
Yes I got the direction confused, I have made it 9 now.

Reply:
Very cool, I did not know there was a hook for this. Thank you, that makes it a lot more reliable.